pkg/utils/utils.go

```diff
@@ -28,6 +28,19 @@ import (
 var (
 	sysBusPci = "/sys/bus/pci/devices"
 	devDir    = "/dev"
+	// getPfEswitchMode returns PF's eswitch mode for the given VF
+	// If device is not VF then it will return its own eswitch mode
+	getPfEswitchMode = func(pciAddr string) (string, error) {
```

Comment:
not a fan of this, any chance we follow the same pattern in netlink as we did in sriovnet ?

Reply:
i know its a bit of refactoring, but its going to make our life easier down the road.